setup.py

```diff
@@ -6,10 +6,35 @@
 
 
 """
-Spyder kernels
+Spyder Kernels
 ==============
 
-Jupyter kernels for the Spyder console
+Provides Jupyter kernels for use with the Spyder IDE's IPython consoles,
+launched either through Spyder itself or in an independent Python session.
+These allow for interactive or file-based execution of Python code in
+different environments inside of Spyder.
+
+Spyder is a powerful scientific environment written in Python, for Python,
+and designed by and for scientists, engineers and data analysts.
+It features a unique combination of the advanced editing, analysis, debugging
+and profiling functionality of a comprehensive development tool with the data
+exploration, interactive execution, deep inspection and beautiful visualization
+capabilities of a scientific package. For more on Spyder, see our website:
+https://www.spyder-ide.org/
+
+To launch a Spyder IPython console and run code in a different environment
```

Comment:
This doesn't need to be part of the package description. Please remove it.

Reply:
It's very specific to how it works and it's going to change when we add our `ipdb` kernel.

Reply:
Okay, removed.